src/babel/babel-plugin.js

```diff
@@ -64,33 +74,27 @@ module.exports = function(babel, options = {}) {
         if (!state.effects) state.effects = new Set()
         if (!state.domains) state.domains = new Set()
 
-        if (t.isIdentifier(path.node.callee)) {
-          if (stores && storeCreators.has(path.node.callee.name)) {
-            const id = findCandidateNameForExpression(path)
-            if (id) {
-              setStoreNameAfter(path, state, id, babel.types, smallConfig)
-              state.stores.add(id.name)
-            }
-          }
-          if (events && eventCreators.has(path.node.callee.name)) {
-            const id = findCandidateNameForExpression(path)
-            if (id) {
-              setEventNameAfter(path, state, id, babel.types, smallConfig)
-              state.events.add(id.name)
-            }
-          }
-          if (effects && effectCreators.has(path.node.callee.name)) {
-            const id = findCandidateNameForExpression(path)
-            if (id) {
-              setEventNameAfter(path, state, id, babel.types, smallConfig)
-              state.effects.add(id.name)
-            }
-          }
-          if (domains && domainCreators.has(path.node.callee.name)) {
-            const id = findCandidateNameForExpression(path)
-            if (id) {
-              setEventNameAfter(path, state, id, babel.types, smallConfig)
-              state.domains.add(id.name)
+        if (t.isIdentifier(path.node.callee) && this.importedNames.has(path.node.callee.name)) {
+          const name = this.importedNames.get(path.node.callee.name)
+          const id = findCandidateNameForExpression(path)
+          if (id) {
+            switch (name) {
+              case definions.createStore:
```

Comment:
why do we even use it here? `definitions` are hardcoded defaults, aren't it?

Reply:
I'm used one common map for all tracked imports related with effector, so because we have different handlers for `createStore`, `createEvent`, `createEffect` and `createDomain`, we need match imported name with its implementation

Reply:
But CI failure implies wrong assumption here: something definitely going wrong https://github.com/zerobias/effector/pull/205/checks?check_run_id=230852210#step:5:101

Reply:
I think its nor related with my changes, i'm rebase from master now, tests snapshots failed only with `sid`s
` PASS   effector/naming  src/effector/naming/__tests__/name.test.js`

Reply:
But tests from CI was still fails hmm

Reply:
Seems babel test snapshot now match with CI, but tests wich use babel-plugin fails (but on my local machine tests pass)